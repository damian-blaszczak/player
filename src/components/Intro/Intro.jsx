import React from 'react';
import { Link } from 'react-router-dom';
import routes from '../../routes.const';
import Title from '../Title/Title';
import styled from 'styled-components';

const IntroContainer = styled.div`
  background: linear-gradient(180deg, rgba(83,64,84,1) 0%, rgba(41,29,60,1) 40%, rgba(41,29,60,1) 100%);
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  padding: 0 20px;
  height: 100vh
`;

const linkStyles = { margin: '0 5px', color: 'grey' };

const Intro = () => (
  <IntroContainer>
      <Title>
        Kliknij
        <Link to={routes.player} style={linkStyles}>
          tutaj
        </Link>
        aby przejść do odtwarzacza
    </Title>
  </IntroContainer>
);

export default Intro;
