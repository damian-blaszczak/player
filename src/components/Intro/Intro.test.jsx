import React from 'react';
import { render } from '@testing-library/react';
import Intro from './Intro';
import { MemoryRouter as Router } from 'react-router-dom';

it('renders correctly', () => {
  const { container } = render(
    <Router>
      <Intro />
    </Router>
  );
  expect(container).toMatchSnapshot();
});
