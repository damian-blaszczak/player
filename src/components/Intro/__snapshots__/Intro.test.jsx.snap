// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="sc-AxirZ bRkqVq"
  >
    <h2
      class="sc-AxjAm ijAwJO"
    >
      Kliknij
      <a
        href="/player"
        style="margin: 0px 5px; color: grey;"
      >
        tutaj
      </a>
      aby przejść do odtwarzacza
    </h2>
  </div>
</div>
`;
